lib/tools/apk-signing.js

```diff
@@ -70,7 +70,14 @@ apkSigningMethods.executeApksigner = async function (args = []) {
     log.warn(`Got an error during apksigner execution: ${err.message}`);
     for (const [name, stream] of [['stdout', err.stdout], ['stderr', err.stderr]]) {
       if (stream) {
-        log.warn(`apksigner ${name}: ${stream}`);
+        log.warn(`apksigner ${name}:`);
```

Comment:
I find the stacktraces useful though

Reply:
Really? Into Android code? Like
```
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/android/apksigner/ApkSignerTool : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
 	at java.lang.ClassLoader.defineClass(ClassLoader.java:803)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:442)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:64)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:354)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:348)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:347)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:312)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:482
```

Reply:
yep. that helped to investigate file access issue